packages/nextjs/src/config/webpack.ts

```diff
@@ -254,7 +254,7 @@ function checkWebpackPluginOverrides(
  * @returns `true` if sentry code should be injected, and `false` otherwise
  */
 function shouldAddSentryToEntryPoint(entryPointName: string): boolean {
-  return entryPointName === 'pages/_app' || entryPointName.includes('pages/api');
+  return entryPointName === 'pages/_app' || entryPointName === 'pages/_error' || entryPointName.includes('pages/api');
```

Comment:
Are there now situations where sentry is injected twice? Will that cause problems?

Reply:
Yes, and no, at least on the server side. Right now, we inject into every API route, so in any case where a server handles more than one API request, we're already running the `sentry.server.config.js` code multiple times. There's a check in the `init` code, so that it will bail if it detects that the SDK is already running: https://github.com/getsentry/sentry-javascript/blob/a338b1b0f823dac931212619961bb5a3293e838a/packages/nextjs/src/index.server.ts#L42-L45

As for client-side, I think it would also be fine, but given that this is solving a problem which as far as I know only happens serverside, I'll restrict it to the server build.